debian/tests/control

```diff
@@ -1,10 +1,31 @@
 # DEP-8 continuous integration tests.
 #
-#  TODO: Add in tree unit testing whenever it becomes available.
+#  TODO: Make integration testing run
 #
 
 Test-Command: nicotine --version | grep Nicotine+
 Depends: nicotine
 Restrictions: isolation-container
 Features: test-name=test-version
 
+Test-Command: pytest-3 tests/unit
+Depends:
+    nicotine,
```

Comment:
You can replace this line with @ per [spec](https://salsa.debian.org/ci-team/autopkgtest/raw/master/doc/README.package-tests.rst).

Reply:
Alright, done.